packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -78,6 +78,9 @@ class ServiceConnectionManager {
 
   final isolateManager = IsolateManager();
 
+  bool get isMainIsolatePaused =>
```

Comment:
It may be helpful to add a JSDoc here stating the default if there is no mainIsolate.